blueprints/codes/formula.py

```diff
@@ -0,0 +1,62 @@
+"""Module for the abstract base class Formula."""
+from abc import ABC, abstractmethod
+
+
+class Formula(float, ABC):
```

Comment:
Good point @SZeltaat but I think we should stick to the mathematical definition because 'Formula' is supposed to be used on different types of codes and not only on the eurocodes.